pyproject.toml

```
@@ -82,3 +148,47 @@ log_format = "%(asctime)s.%(msecs)03d %(levelname)-8s %(name)-12s %(filename)s:%
 log_date_format = "%Y-%m-%dT%H:%M:%S"
 log_cli_format = "%(asctime)s.%(msecs)03d %(levelname)-8s %(name)-12s %(filename)s:%(lineno)s:%(funcName)s %(message)s"
 log_cli_date_format = "%Y-%m-%dT%H:%M:%S"
+
+[tool.isort]
+profile = "black"
+py_version = 37
+line_length = 88
+src_paths= ["rdflib", "test"]
+supported_extensions = ["pyw", "pyi", "py"]
+skip = [
+    '.eggs',         # exclude a few common directories in the
+    '.git',          # root of the project
+    '.hg',
+    '.mypy_cache',
+    '.pytest_cache',
+    '.tox',
+    '.venv',
+    '.github',
+    '_build',
+    'htmlcov',
+    'benchmarks',
+    'examples',       # No need to Black examples
+    'test_reports',
+    'rdflib.egg-info',
+    'buck-out',
+    'build',
+    'dist',
+    'venv',
+]
+
+[tool.mypy]
+files = ['rdflib', 'test', 'devtools']
+python_version = "3.7"
+warn_unused_configs = true
+ignore_missing_imports = true
+disallow_subclassing_any = false
+warn_unreachable = true
+warn_unused_ignores = true
+no_implicit_optional = false
+
+[[tool.mypy.overrides]]
+module = "pyparsing.*"
+# This is here because of an upstream issue with pyparsing:
+# https://github.com/pyparsing/pyparsing/issues/385
+# Once the issue is fixed this should be removed.
+follow_imports = "skip"
```

You also need a `build-system` section:

```suggestion
follow_imports = "skip"

[build-system]
requires = ["poetry-core"]
build-backend = "poetry.core.masonry.api"
```

---

Added now.